### PR TITLE
libretro-fsuae: add cross-compilation compatibility.

### PR DESCRIPTION
**Pull request by repojohnray**
game.libretro.fsuae: updated to 'any' architecture.

### CODE REVIEW

**Comment by CvH** [comment]
@5schatten ^^^^

File: packages/emulation/libretro-fsuae/package.mk [hunk 1]
```diff
@@ -28,7 +28,8 @@ pre_configure_target() {
 }
 
 make_target() {
-  make -j1 CC="$CC" gen
+  make -j1 CC="$TOOLCHAIN/bin/host-gcc" CPPFLAGS="$HOST_CPPFLAGS" CFLAGS="$HOST_CFLAGS" LDFLAGS="$HOST_LDFLAGS" gen
```
Comment (vpeter4) [inline packages/emulation/libretro-fsuae/package.mk]: `$TOOLCHAIN/bin/host-gcc` == `$HOST_CC`

Comment (repojohnray) [reply packages/emulation/libretro-fsuae/package.mk]: Replaced by $HOST_CC

Commit updated to 21d5c1a97ed558fb2c578a1b7e3da4c1e609a5f9

Comment (repojohnray) [reply packages/emulation/libretro-fsuae/package.mk]: Commit updated.

**Comment by CvH** [comment]
sorry for **very** late reply
@5schatten pointed out (intern, weeks ago ... ups) that it makes no sense (usage wise) to build `libretro-fsuae` for anything else then x86_64, I already forgot the details again but it was a rather fundamental NO.

**Comment by repojohnray** [comment]
FSUAE is not optimized for small device, but the speed of a today ARM board is enough to emulate an A500 (accuracy = 0) using libretro-fsuae. And, this will improve over time.

We have at least two working projects on different sources baseline: libretro-uae and libretro-fsuae, having a choice is likely helpful.

**Comment by 5schatten** [comment]
Well the latest libretro-fsuae is based on FS-UAE 2.8.1u5 which is rather dated and the developement is not that active. AFAIK FS-UAE was never optimized for ARM as stated here https://fs-uae.net/docs/faq/comment-page-4#comment-938732

There is reason why [Amiberry ](https://github.com/midwan/amiberry) developement was focused on ARM https://github.com/midwan/amiberry/issues/131#issuecomment-336399780

But well if it works and somebody uses it feel free to do so  ¯\_(ツ)_/¯

**Comment by repojohnray** [comment]
The patch is still available if someone want to test it.

**Comment by MilhouseVH** [comment]
@repojohnray Closing this (with thanks) as it doesn't sound like this change is appropriate for LibreELEC at this time, but if any downstream projects want to enable FS-UAE for arm then this patch will always be here.

**Pull request closed without merging**